python-sdk/src/astro/files/locations/google/gcs.py

```diff
@@ -43,6 +43,21 @@ def size(self) -> int:
             object_name = object_name[1:]
         return int(self.hook.get_size(bucket_name=bucket_name, object_name=object_name))
 
+    def databricks_settings(self) -> dict:
```

Comment:
Should this be changed to `databricks_auth_settings` on all file locations?

Reply:
@kaxil I like this idea.